src/sentry/static/sentry/app/components/events/interfaces/frame/frame.tsx

```diff
@@ -18,14 +18,37 @@ import {SymbolicatorStatus} from 'app/components/events/interfaces/types';
 import {combineStatus} from 'app/components/events/interfaces/debugMeta/utils';
 import {IconRefresh, IconAdd, IconSubtract, IconQuestion, IconWarning} from 'app/icons';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Frame as FrameType, SentryAppComponent, PlatformType} from 'app/types';
```

Comment:
Sometimes I would like to do that too, but I was told that I shouldn't do it..maybe we should bring this topic to the TSC meeting 🤓

```suggestion
import {Frame, SentryAppComponent, PlatformType} from 'app/types';
```

Reply:
@priscilawebdev Well, the problem is that the component here is named `Frame`. Unless I rename the component, I can't import the type as `Frame`.

Reply:
I think this is fine since you're converting to tsx -- would be good to follow up and rename the type though (I imagine that's why we have `PlatformType`)